### DISCUSSION TITLE
Declaration attribute: visibility missing

### DISCUSSION THREAD

**Discussion by renlite** [c3lang/c3c#1183] [Q&A]
https://github.com/orca-app/orca/blob/main/src/platform/platform.h#L105 defines for callbacks:
```C
#define ORCA_EXPORT __attribute__((visibility("default")))

ORCA_EXPORT void oc_on_init(void){ ... }

```
I think it's needed for C-binding.
I didn't find a proper macro (declaration attribute) in C3. Is this missing or can I create a user defined attribute?
```C
def @MyAttribute = {  __attribute__((visibility("default"))) };
```

**Comment by lerno** [comment]
It has the default visibility, so you should be able to ignore it completely, so declare for example

`extern fn void oc_on_init();`
